libraries/program-error/tests/into.rs

```diff
@@ -0,0 +1,22 @@
+//! Tests `#[derive(IntoProgramError)]`
+//!
+use spl_program_error::*;
+
+/// Example error
+#[derive(
+    Clone, Debug, Eq, IntoProgramError, thiserror::Error, num_derive::FromPrimitive, PartialEq,
+)]
+pub enum ExampleError {
+    /// Mint has no mint authority
+    #[error("Mint has no mint authority")]
+    MintHasNoMintAuthority,
+    /// Incorrect mint authority has signed the instruction
+    #[error("Incorrect mint authority has signed the instruction")]
+    IncorrectMintAuthority,
+}
+
+/// Tests that all macros compile
+#[test]
+fn _test() {
```

Comment:
nit: same here, may as well just call this `test_macros_compile`. side note: I'm not a fan of underscore prefacing when it's not necessary